src/pages/Donate/Content.tsx

```diff
@@ -56,10 +56,7 @@ function Content(props: Props) {
       />
       {/** small screen but space is still enough to render sidebar */}
       <div className="mx-0 border-b md:contents min-[445px]:border min-[445px]:mx-4 rounded border-prim">
-        <Steps
-          className="md:border border-prim col-start-1 md:col-start-2 md:row-start-2 md:row-span-2 lg:row-span-1"
-          donaterConfig={null}
```

Comment:
`donaterConfig = null` is explicitly saying - "widget config is not needed when rendering this component in `/donate`"

changing this to likes of "widget config is optional in this page.." removes that extra clarity. 

also widget config is explicit to internal components as it drives donation source. 
`const donationSource = config === null ? "bg-marketplace" : "bg-widget"`

When I factored out splits from the donate forms, It errored out when I deleted donaterConfig (drives advanced options /splits) as it is still needed to determine donation source.

Reply:
Setting `donaterConfig={null}` conveys the same intent as completely omitting it (the intent being "the prop is not desired"). The former only forces you to convey the intent in more characters. 

I guess this is just a matter of preference, so will revert the change so as not to hold up the whole PR for a trifle.

Reply:
if this is just preference, I would have not commented on this. There's a big difference between leaving config out (optional) and explicitly specifying that config is not need (`null`) - former doesn't complain 
<img width="418" alt="image" src="https://github.com/AngelProtocolFinance/angelprotocol-web-app/assets/89639563/ef27d4c9-3e8f-4cf3-9cd5-45079023a38b">

while latter forces you to be explicit on your intent
<img width="795" alt="image" src="https://github.com/AngelProtocolFinance/angelprotocol-web-app/assets/89639563/f08b1a1e-7117-41e4-9868-b7b19e46ebaa">

just imagine using component inside widget and didn't specify config (no error - it's optional anyway) - suddenly widget donations are recorded as marketplace donations.